README.md

```diff
@@ -1,78 +1,66 @@
 # Apify Keboola Extractor
 
-Apify extractor for Keboola Connection
+You can use it to connect data from [Apify platform](https://apify.com/) into [Keboola platform](https://www.keboola.com/).
 
-## Test
+## Resources
 
-### Sample configuration
+* [Apify documentation](https://docs.apify.com/tutorials/integrations#get-started)
+* [Getting started tutorial](https://help.apify.com/en/articles/2003234-keboola-integration)
+* [Apify component](https://components.keboola.com/components/apify.apify) page on Keboola developer platform
+* [Keboola documentation for developers](https://developers.keboola.com/overview/)
+* [Keboola API blueprint](https://kebooladocker.docs.apiary.io/#reference/actions/run-custom-component-action/process-action)
 
-Before run, you need to have one of configuration in `path_to_data_folder` folder.
+----------
 
-#### To run a crawler
+## Local Development
 
-```
-{
-  "parameters": {
-    "userId": "Apify user ID",
-    "#token": "Apify API token"
-    "crawlerId": "H8Xo8nhNDHJkmEfJG",
-    "crawlerSettings": {}
-  }
-}
-```
+If you are interested in adding a new feature or fixing a bug in the integration, feel free to open a pull request.
 
-#### To get results from existing execution
+Before you started developing this integration, you will need your Apify API token and user ID.
+You can find the token and ID [on the Integrations page of your Apify account](https://my.apify.com/account#/integrations).
```

Comment:
started => start

And please also add a note where to get the user ID, just to be sure :)